redfish-core/lib/systems.hpp

```diff
@@ -2076,6 +2076,88 @@ inline void getEnabledPanelFunctions(
         });
 }
 
+/**
+ * Execute a Panel Enabled Function
+ */
+inline void
+    executePanelFunction(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+                         const uint8_t funcNo)
+{
+    BMCWEB_LOG_DEBUG << "Execute Panel function";
+
+    crow::connections::systemBus->async_method_call(
+        [asyncResp](const boost::system::error_code& ec,
+                    const std::tuple<bool, std::string, std::string>& result) {
+        if (ec)
+        {
+            BMCWEB_LOG_ERROR << "Execute a panel function D-bus error:  "
+                             << ec.value();
+            messages::internalError(asyncResp->res);
+            return;
+        }
+
+        if (!std::get<0>(result))
+        {
+            BMCWEB_LOG_ERROR << "ExecutePanelFunction is failed";
```

Comment:
I think including the funcNo in the ERROR output would help with debugging a failure.

Reply:
Done